package.json

```diff
@@ -30,9 +31,9 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-eslint": "^3.0.2",
     "ember-cli-version-checker": "^1.3.1",
+    "exists-sync": "0.0.4",
```

Comment:
I thought `fs.existsSync` is deprecated. The readme of exists-sync says this: https://github.com/ember-cli/exists-sync#exists-sync